<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Configurator</class>
 <widget class="QDialog" name="Configurator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>475</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(32, 43, 47);</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="pushButtonReset_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>420</y>
     <width>401</width>
     <height>26</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonReset">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>441</width>
     <height>26</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Restore defaults</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelVersion">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>450</y>
     <width>46</width>
     <height>18</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>version</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>261</width>
     <height>162</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">color: rgb(255,255,255);font-size:20px</string>
      </property>
      <property name="text">
       <string>Accuracy Settings</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="labelScreenshots">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Screenshots per second</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QComboBox" name="comboBoxScreenshots">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="currentText">
       <string>15</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>15</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>25</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelUpdates">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Updates per second</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelSecondsInCombat">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Seconds in combat</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QComboBox" name="comboBoxUpdates">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="2" colspan="2">
     <widget class="QComboBox" name="comboBoxSecondsInCombat">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="labelConsideredLines">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string># of Lines in Combat Log</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="2" colspan="2">
     <widget class="QComboBox" name="comboBoxConsideredLines">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string>10</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>24</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>27</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>203</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">color: rgb(255,255,255);font-size:20px</string>
      </property>
      <property name="text">
       <string>General</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkBoxToolbar">
      <property name="styleSheet">
       <string notr="true">color:white;font-size:14px;</string>
      </property>
      <property name="text">
       <string>Show Toolbar</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBoxDetails">
      <property name="styleSheet">
       <string notr="true">color:white;font-size:14px;</string>
      </property>
      <property name="text">
       <string>Show Details</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="checkBoxExtraDetails">
      <property name="styleSheet">
       <string notr="true">color:white;font-size:14px;</string>
      </property>
      <property name="text">
       <string>Show Additional Details</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="checkBoxOpacity">
      <property name="styleSheet">
       <string notr="true">color:white;font-size:14px;</string>
      </property>
      <property name="text">
       <string>Toggle Opacity</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="checkBoxOBS">
      <property name="styleSheet">
       <string notr="true">color:white;font-size:14px;</string>
      </property>
      <property name="text">
       <string>Enable OBS Mode</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QComboBox" name="comboBoxLanguage">
      <item>
       <property name="text">
        <string>en</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>de</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>fr</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>271</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item row="8" column="0">
     <widget class="QCheckBox" name="checkBoxPerDmg">
      <property name="styleSheet">
       <string notr="true">color:white;font-size:14px;</string>
      </property>
      <property name="text">
       <string>Percental Damage</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="checkBox5sDPS">
      <property name="styleSheet">
       <string notr="true">color:white;font-size:14px;</string>
      </property>
      <property name="text">
       <string>Show Last5sDPS</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">color: rgb(255,255,255);font-size:20px</string>
      </property>
      <property name="text">
       <string>Player Bar Settings</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkBoxProfColors">
      <property name="styleSheet">
       <string notr="true">color:white;font-size:14px;</string>
      </property>
      <property name="text">
       <string>Show Profession</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBoxName">
      <property name="styleSheet">
       <string notr="true">color:white;font-size:14px;</string>
      </property>
      <property name="text">
       <string>Show Name</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="checkBoxDPS">
      <property name="styleSheet">
       <string notr="true">color:white;font-size:14px;</string>
      </property>
      <property name="text">
       <string>Show DPS</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="checkBoxDamage">
      <property name="styleSheet">
       <string notr="true">color:white;font-size:14px;</string>
      </property>
      <property name="text">
       <string>Show Damage</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QCheckBox" name="checkBoxPosition">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;font-size:14px;</string>
      </property>
      <property name="text">
       <string>Show Position</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QCheckBox" name="checkBoxCDPS">
      <property name="styleSheet">
       <string notr="true">color:white;font-size:14px;</string>
      </property>
      <property name="text">
       <string>(Show CondiDPS)</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="groupOnly">
      <property name="styleSheet">
       <string notr="true">color:white;font-size:16px;</string>
      </property>
      <property name="text">
       <string>Group only:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QCheckBox" name="checkBoxCDamage">
      <property name="styleSheet">
       <string notr="true">color:white;font-size:14px;</string>
      </property>
      <property name="text">
       <string>(Show CondiDmg)</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QCheckBox" name="checkBoxCPerDmg">
      <property name="styleSheet">
       <string notr="true">color:white;font-size:14px;</string>
      </property>
      <property name="text">
       <string>(Percental Condi)</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QCheckBox" name="checkBoxActivity">
      <property name="styleSheet">
       <string notr="true">color:white;font-size:14px;</string>
      </property>
      <property name="text">
       <string>(Show Activity)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>281</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">color: rgb(255,255,255);font-size:20px</string>
      </property>
      <property name="text">
       <string>Graph Settings</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkBoxGraphShow">
      <property name="styleSheet">
       <string notr="true">color:white;font-size:14px;</string>
      </property>
      <property name="text">
       <string>Show Graph</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBoxGraphAvgDPS">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Green Graph&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;font-size:14px;</string>
      </property>
      <property name="text">
       <string>Average PersDPS</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="checkBoxGraphAvgGroupDPS">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Yellow Graph&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;font-size:14px;</string>
      </property>
      <property name="text">
       <string>Average GrDPS</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="checkBoxGraphAvgCDPS">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Purple Graph&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;font-size:14px;</string>
      </property>
      <property name="text">
       <string>Average CondiDPS</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="checkBoxGraph5sDPS">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Red Graph&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;font-size:14px;</string>
      </property>
      <property name="text">
       <string>Last5sDPS</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QCheckBox" name="checkBoxGraphRealDPS">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Blue Graph&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;font-size:14px;</string>
      </property>
      <property name="text">
       <string>Show RealDPS</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>210</y>
     <width>281</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">color: rgb(255,255,255);font-size:20px</string>
      </property>
      <property name="text">
       <string>Rebind Hotkeys</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelShortcut1">
      <property name="styleSheet">
       <string notr="true">color:white;font-size:14px;</string>
      </property>
      <property name="text">
       <string>Toggle CombatMode</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelShortcut2">
      <property name="styleSheet">
       <string notr="true">color:white;font-size:14px;</string>
      </property>
      <property name="text">
       <string>Reset Shortcut</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="labelShortcut3">
      <property name="styleSheet">
       <string notr="true">color:white;font-size:14px;</string>
      </property>
      <property name="text">
       <string>GlobalReset Shortcurt</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QKeySequenceEdit" name="seqEditGlobalReset"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="labelShortcut4">
      <property name="styleSheet">
       <string notr="true">color:white;font-size:14px;</string>
      </property>
      <property name="text">
       <string>Save Log Shortcut</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QKeySequenceEdit" name="seqEditCombatMode"/>
    </item>
    <item row="2" column="2">
     <widget class="QKeySequenceEdit" name="seqEditReset"/>
    </item>
    <item row="4" column="2">
     <widget class="QKeySequenceEdit" name="seqEditSave"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonReset_2</sender>
   <signal>clicked()</signal>
   <receiver>Configurator</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
